yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -155,6 +155,29 @@ func saveMigrationReportsFn(msr *metadb.MigrationStatusRecord) {
 		}
 	}
 
+	askAndStorePasswords(msr)
+	passwordsEnvVars := []string{
+		fmt.Sprintf("TARGET_DB_PASSWORD=%s", targetDBPassword),
+		fmt.Sprintf("FF_DB_PASSWORD=%s", fallForwardDBPassword),
+		fmt.Sprintf("SOURCE_DB_PASSWORD=%s", sourceDBPassword),
+	}
+
+	if msr.ExportType == "snapshot-and-changes" { // streaming changes case
+		utils.PrintAndLog("save live migration reports...")
+		liveMigrationReportFilePath := filepath.Join(backupDir, "reports", "live_migration_report.txt")
+		strCmd := fmt.Sprintf("yb-voyager live-migration report -e %s > %q", exportDir, liveMigrationReportFilePath)
+		liveMigrationReportCmd := exec.Command("bash", "-c", strCmd)
+		liveMigrationReportCmd.Env = append(os.Environ(), passwordsEnvVars...)
+		var outbuf bytes.Buffer
+		liveMigrationReportCmd.Stderr = &outbuf
+		err = liveMigrationReportCmd.Run()
+		if err != nil {
+			log.Errorf("end migration: running live migration report command: %s: %v", outbuf.String(), err)
+			utils.ErrExit("end migration: running live migration report command: %v", err)
+		}
+		return
```

Comment:
After saving, let's print something like "Saved data migration report at <filepath>" ?

Reply:
we can do that. If we do that we should also do the same for other backup flags(=true) also.
But it will cluster the output for end migration command.

What do you think is it really required considering we will be having a well-structured `backup-dir` like we have for `export-dir`? @makalaaneesh

Reply:
In that case, we can have a single message at the end mentioning what all we have backed up and where, what say?

Reply:
yeah makes sense, let me add that.